src/Sdk/Common/Common/Authentication/NoOpCredentials.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace GitHub.Services.Common
+{
+    // Set of classes used to bypass token operations
+    // Results Service and External services follow a different auth model but
+    // we are required to pass in a credentials object to create a RawHttpMessageHandler
+    public class NoOpCredentials : FederatedCredential
```

Comment:
do you need to have `FedCred` or just use `IssuedTokenCredential`?

Reply:
then you can always return false for `IsAuthenticationChallenge()`

Reply:
FedCred, it's on the base constructor.  

```
        public RawHttpMessageHandler(
            FederatedCredential credentials,
            RawClientHttpRequestSettings settings,
            HttpMessageHandler innerHandler)
```